### DISCUSSION TITLE
ok sorry for this coming late 
ok 
I decided to use wazuh has my project topic 

so I was asked different questions 
about the methodology of wazuh 

even ask about the data flow diagram, use case diagram and flowchart diagram

### DISCUSSION THREAD

**Discussion by brighthart3333** [wazuh/wazuh#14421] [Q&A] [answered]
I really need to know the wazuh 
data flow diagram 
use case diagram 
flowchart diagram

**Comment by xrisbarney** [comment, marked as answer]
Hi @brighthart3333 

Thank you for using Wazuh!

Wazuh is an open source security platform with XDR and SIEM protection for endpoints and cloud workloads.

Our documentation contains all the information about Wazuh and its architecture. The documentation can be found [here](https://documentation.wazuh.com/current/index.html).

To be specific, the following links would help in your project:

- Component data flow diagram documentation is [here](https://documentation.wazuh.com/current/getting-started/components/index.html)
- Use cases can be found [here] (https://documentation.wazuh.com/current/getting-started/use-cases/index.html) and [here] (https://documentation.wazuh.com/current/learning-wazuh/index.html)
- The Wazuh architecture can be found [here](https://documentation.wazuh.com/current/getting-started/architecture.html)
- You can deploy Wazuh quickly using the Quickstart guide [here](https://documentation.wazuh.com/current/quickstart.html)

Please let me know if this helps, and Good luck with your project!

**Reply by brighthart3333** [reply]
thanks  I will give you a feedback afterwards how can I send you a word document of my work so you can go through it?

**Reply by xrisbarney** [reply]
You do not need to send a copy of your project to us for review. Since it is academic work, your supervisor will be better placed to review it.

however, if you run into any troubles, feel free to ask questions here or on our slack channel https://join.slack.com/t/wazuh/shared_invite/zt-1bxio6ks1-~HaIHEYlWqlVsNUBovkjUA.

**Reply by brighthart3333** [reply]
thanks 
can I know the methodology used in wazuh

**Reply by xrisbarney** [reply]
Hi @brighthart3333 

What do you mean by methodology?

Do you mean the development process? For the development process, Wazuh is an open source security platform, you can check out our GitHub repository [here](https://github.com/wazuh). We work on improving the solution in order to give users a great out-of-the-box experience with Wazuh. We also accept contributions and pull requests from the community for product improvements.

The Wazuh solution is composed of three [central platform components](https://documentation.wazuh.com/current/getting-started/components/index.html) that can be deployed on one server or distributed servers and a single [agent](https://documentation.wazuh.com/current/installation-guide/wazuh-agent/index.html) on the endpoint.

The [Wazuh server](https://documentation.wazuh.com/current/getting-started/components/wazuh-server.html) analyzes data received from the agents. It processes it through decoders and rules, using threat intelligence to look for well-known indicators of compromise (IOCs). This component is also used to manage the agents, configuring and upgrading them remotely when necessary.

The Wazuh agent is multi-platform and runs on the hosts the user wants to monitor. It communicates with the Wazuh manager, sending data in near real-time through an encrypted and authenticated channel.

**Reply by brighthart3333** [reply]
I appreciate this 
I was also looking through the wazuh use case 


I dont know I can form the use case diagram of wazuh
cause I just want the to know the Use case diagram of wazuh

**Reply by xrisbarney** [reply]
you can use this image below to illustrate how Wazuh components interact with each other and the different possible use cases on the endpoint.
![image](https://user-images.githubusercontent.com/25141115/181230878-f32aa692-3395-41dd-b59e-c24a27d3c2b7.png)
